xbmc/powermanagement/PowerManager.cpp

```diff
@@ -246,16 +247,28 @@ void CPowerManager::OnSleep()
   if (dialog)
     dialog->Open();
 
-  CLog::Log(LOGNOTICE, "%s: Running sleep jobs", __FUNCTION__);
+  CLog::Log(LOGNOTICE, "CPowerManager::OnSleep(): Running sleep jobs");
 
   // stop lirc
 #if defined(HAS_LIRC) || defined(HAS_IRSERVERSUITE)
-  CLog::Log(LOGNOTICE, "%s: Stopping lirc", __FUNCTION__);
+  CLog::Log(LOGNOTICE, "CPowerManager::OnSleep(): Stopping lirc");
   CBuiltins::GetInstance().Execute("LIRC.Stop");
 #endif
 
   CServiceBroker::GetPVRManager().OnSleep();
-
+  if (g_application.m_pPlayer->IsPlaying())
+  {
+    m_lastFileItem.reset(new CFileItem(g_application.CurrentFileItem()));
+    //keep the actual offset instead of store and load it from database
```

Comment:
we usually put a space between `//`and the comment's text

Reply:
Since my recent proposed changes re. stacking (PR #13235) CurrentFileItem() will return the stack, both in case of ISO and Regular stack. Calling PlayFile like you do in OnWake will work correctly now, because PlayFile calls PlayStack with the correct setting.